storybookjs/storybook
# Remove upper bound on react-native peerDependency

Issue: #1418

The goal is to allow new `react-native` versions without having to maintain an upper bounded version in Storybook. 

## What I did

Updates Storyboard's `react-native` module to allow any versions `>=0.27.0`, rather than keeping a capped upper-bound. As described in https://github.com/storybooks/storybook/issues/1418#issuecomment-313234328, if a breaking change is later introduced the next version of storyboard can set the new minimum version. This removes the need to maintain an upper bound every time `react-native` rotates out a new release. I personally think it's okay to be a little less strict here.



_Note: we cannot use `^0.27.0` since this is a prerelease, which will not be satisfied by semver_

## How to test

- Create a new `react-native` project on a version later than `0.43.x`
- Locally link `@storybook/react-native` and observe a successful installation without an invalid `peerDependency` message
